indico/modules/events/timetable/controllers/display.py

```diff
@@ -100,7 +103,69 @@ def _process(self):
                                 back_url=url_for('.timetable', self.event))
 
 
+class RHTimetableExportWeasyPrint(RHTimetableProtectionBase):
+    def _process(self):
+        form = TimetablePDFExportForm(formdata=request.args, csrf_enabled=False)
+        if form.validate_on_submit():
+            from flask import render_template
+            from indico.modules.events.timetable.models.entries import TimetableEntry
+            from indico.core.db import db
+            from pathlib import Path
```

Comment:
I guess this is just for testing, but just in case: We only use local imports when it's needed to avoid errors with circular imports